CleverTap/Runtime/Native/UnityNativeWrapper/UnityNativeNetworkEngine.cs

```diff
@@ -334,6 +341,46 @@ private void ApplyNetworkEngineRequestConfiguration(UnityNativeRequest request)
             }
         }
 
+        private Dictionary<string, string> AddARP(Dictionary<string, string> headers)
+        {
+            string arpNamespaceKey = string.Format(UnityNativeConstants.Network.ARP_NAMESPACE_KEY,
+                _coreState.AccountInfo.AccountId, _coreState.DeviceInfo.DeviceId);
+
+            if (string.IsNullOrEmpty(arpNamespaceKey))
```

Comment:
`arpNamespaceKey` cannot be null or empty

Reply:
removed the check